tests/h/search/query_test.py

```diff
@@ -36,174 +48,97 @@ class TestBuilder(object):
         ("32.7", OFFSET_DEFAULT),
         ("9801", OFFSET_MAX),
     ])
-    def test_offset(self, offset, from_):
-        builder = query.Builder(ES_VERSION)
+    def test_offset(self, pyramid_request, offset, from_):
+        limiter = query.Limiter()
+        search = elasticsearch_dsl.Search(
+            using="default", index=pyramid_request.es.index
+        )
 
+        params = {"offset": offset}
         if offset is MISSING:
-            q = builder.build({})
-        else:
-            q = builder.build({"offset": offset})
+            params = {}
+
+        q = limiter(search, params).to_dict()
```

Comment:
👍 - Good to have a few examples of testing without having to execute an ES query.